libs/gorules/rule_empty_slices.go

```diff
@@ -0,0 +1,14 @@
+package gorules
+
+import "github.com/quasilyte/go-ruleguard/dsl"
+
+// DeclaringEmptySlices detects empty slice declarations and suggests using nil slices.
+func DeclaringEmptySlices(m dsl.Matcher) {
```

Comment:
Please include a link to https://go.dev/wiki/CodeReviewComments#declaring-empty-slices